aspnetcore/data/ef-rp/complex-data-model.md

```diff
@@ -571,10 +571,17 @@ The ALTER TABLE statement conflicted with the FOREIGN KEY constraint "FK_dbo.Cou
 database "ContosoUniversity", table "dbo.Department", column 'DepartmentID'.
 ```
 
-When migrations are run with existing data, there may be FK constraints that are not satisfied with the exiting data. For this tutorial, a new DB is created, so there are no FK constraint violations. See
-[Fixing foreign key constraints with legacy data](#fk) for instructions on how to fix the FK violations on the current DB.
+When migrations are run with existing data, there may be FK constraints that are not satisfied with the existing data. For this tutorial, a new DB is created, so there are no FK constraint violations. See [Apply the migration to the existing database](#applyexisting) for instructions on how to fix the FK violations on the current DB.
 
-### Drop and update the database
+## Apply the migration
+
+Now that you have an existing database, you need to think about how to apply future changes to it. For this, you have two approaches:
+* [Drop and re-create the database](#drop)
+* [Apply the migration to the existing database](#applyexisting). While this method more complex and time-consuming, it's the preferred approach for real-world, production environments. 
```

Comment:
Consider making it clearer that in this tutorial the choices are mutually exclusive:

This is an optional section of the tutorial. You can do the drop and re-create steps and skip this section. If you do want to follow the steps in this section, don't do the drop and re-create steps.

missing "is" before "more complex"

Reply:
On re-reading this I have too many "do"s in a row; feel free to reword, e.g., by eliminating "do" before "want" or some other rewording.